newIDE/app/src/EffectsList/index.js

```diff
@@ -486,213 +688,184 @@ export default function EffectsList(props: Props) {
-                                            ),
-                                        },
-                                      ]}
-                                    />
-                                    <Spacer />
-                                  </div>
-                                  {effectType && (
-                                    <Line expand noMargin>
-                                      <Column expand>
-                                        {effectMetadata ? (
-                                          <React.Fragment>
-                                            <Line>
-                                              <BackgroundText>
-                                                <MarkdownText
-                                                  source={
-                                                    effectMetadata.description
-                                                  }
-                                                />
-                                              </BackgroundText>
-                                            </Line>
-                                            <PropertiesEditor
-                                              instances={[effect]}
-                                              schema={
-                                                effectMetadata.parametersSchema
-                                              }
-                                              project={props.project}
-                                              resourceManagementProps={
-                                                props.resourceManagementProps
-                                              }
-                                              renderExtraDescriptionText={
-                                                showEffectParameterNames
-                                                  ? parameterName =>
-                                                      i18n._(
-                                                        t`Parameter name in events: \`${parameterName}\` `
-                                                      )
-                                                  : undefined
-                                              }
-                                            />
-                                          </React.Fragment>
-                                        ) : null}
-                                        <Spacer />
-                                      </Column>
-                                    </Line>
-                                  )}
-                                </div>
-                              )
-                            }
-                          </DragSourceAndDropTarget>
-                        );
-                      }
-                    )}
+                {props.layerRenderingType !== '2d' && (
+                  <Column>
+                    <Column>
```

Comment:
I would add a noMargin here to keep the title and the effects aligned
<img width="32" alt="image" src="https://github.com/4ian/GDevelop/assets/32449369/1ce786de-5102-4734-a912-36fa9071ab43">